### DISCUSSION TITLE
French custom sentence

### DISCUSSION THREAD

**Discussion by faduchesne** [music-assistant/support#2460] [Q&A]
Hi, Does anyone know the custom sentence in French for controlling  Music Assistant with Assist? Thank you for your help.

**Comment by OzGav** [comment]
My test agent returns the correct response to `jouer au panama par van halen sur le lecteur d'étude` (Thank you Google translate!) but the intent matching may not recognise that. Let me ask around

**Reply by faduchesne** [reply]
Thx for your help !

**Reply by OzGav** [reply]
Sorry I wasn't able to get back to you. We are likely going to switch to a custom sentence option soon so then you will able to customise in French.
